pkg/kotsadmsnapshot/backup.go

```diff
@@ -977,18 +982,34 @@ func excludeShutdownPodsFromBackup(ctx context.Context, clientset kubernetes.Int
 		"status.phase": string(corev1.PodFailed),
 	}
 
-	podListOption := metav1.ListOptions{
-		LabelSelector: labels.SelectorFromSet(veleroBackup.Spec.LabelSelector.MatchLabels).String(),
-		FieldSelector: fields.SelectorFromSet(selectorMap).String(),
+	labelSets := []string{}
+	if veleroBackup.Spec.LabelSelector.MatchLabels != nil {
```

Comment:
Would this ever be `nil` given that it's initialized here? https://github.com/replicatedhq/kots/pull/4583/files#diff-5bf9dc41509334fa70fd5f20b3c75acac8ac8efa3b3ad54d1d18aa5ee2999792R1051

Reply:
maybe use `len`?

Reply:
oof, will do

Reply:
done